hubble/Makefile

```diff
@@ -0,0 +1,58 @@
+GO := go
+GO_BUILD_FLAGS =
+GO_TEST_FLAGS =
+GO_BUILD = CGO_ENABLED=0 $(GO) build $(GO_BUILD_FLAGS)
+SUBDIRS_HUBBLE_CLI := .
+TARGET=hubble
+VERSION=$(shell cat ../VERSION)
+# homebrew uses the github release's tarball of the source that does not contain the '.git' directory.
+GIT_BRANCH = $(shell command -v git >/dev/null 2>&1 && git rev-parse --abbrev-ref HEAD 2> /dev/null)
+GIT_HASH = $(shell command -v git >/dev/null 2>&1 && git rev-parse --short HEAD 2> /dev/null)
+GO_TAGS ?=
+GOOS ?=
+GOARCH ?=
+
+# Add the ability to override variables
+-include Makefile.override
+
+all: hubble
+
+hubble:
+	$(MAKE) -C $(SUBDIRS_HUBBLE_CLI) hubble-bin
+
+hubble-bin:
+	GOOS=$(GOOS) GOARCH=$(GOARCH) $(GO_BUILD) $(if $(GO_TAGS),-tags $(GO_TAGS)) -ldflags "-w -s -X 'github.com/cilium/cilium/hubble/pkg.GitBranch=${GIT_BRANCH}' -X 'github.com/cilium/cilium/hubble/pkg.GitHash=$(GIT_HASH)' -X 'github.com/cilium/cilium/hubble/pkg.Version=${VERSION}'" -o $(TARGET) $(SUBDIRS_HUBBLE_CLI)
```

Comment:
why the `-o hubble-bin`?

Reply:
it's copied from https://github.com/cilium/hubble/blob/main/Makefile#L43